src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1422,3 +1422,50 @@ function random_orientation_dag(
     end
     return g2
 end
+
+"""
+    bernoulli_graph(Λ)
+
+Given the parametric symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with  probability ``\\operatorname{Bernoulli}(\\Lambda[i,j])``.
+"""
+function bernoulli_graph(
+    Λ::Matrix{Float64};
+    rng::Union{Nothing,AbstractRNG}=nothing,
+    seed::Union{Nothing,Integer}=nothing,
+)
+    n = size(Λ)[1]
```

Comment:
It's good to check that assumptions of a function are satisfied with assertions
```suggestion
    @assert size(Λ)[1] == size(Λ)[2] "The probability matrix must be a square matrix!"
    n = size(Λ)[1]
```